gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/KubernetesExtension.java

```diff
@@ -617,6 +621,14 @@ public String getLogPodNameOrNull() {
     return getOrDefaultString("jkube.log.pod", this::getLogPodName, null);
   }
 
+  public String getLogDateOrNull() {
+    return getOrDefaultString("jkube.docker.logDate", this::getLogDate, null);
```

Comment:
It's consistent with the property/element name in the Maven Plugin and now with the DSL property in Gradle. 
This might come from historic reasons. Some of these configs where inherited from the Docker Maven Plugin (DMP) thus the properties have some sort of DMP naming convention. However the element is descriptive of the config.